amf-api-contract/shared/src/main/scala/amf/apicontract/internal/validation/shacl/ViolationModelValidationPlugin.scala

```diff
@@ -1,25 +1,30 @@
 package amf.apicontract.internal.validation.shacl
 
-import amf.core.client.common.validation.SeverityLevels
+import amf.apicontract.internal.validation.plugin.BaseApiValidationPlugin
+import amf.core.client.common.validation.{ProfileName, SeverityLevels}
 import amf.core.client.common.{NormalPriority, PluginPriority}
 import amf.core.client.scala.model.document.BaseUnit
 import amf.core.client.scala.validation.{AMFValidationReport, AMFValidationResult}
-import amf.core.internal.plugins.validation.{AMFValidatePlugin, ValidationInfo, ValidationOptions, ValidationResult}
+import amf.core.internal.plugins.validation.{ValidationOptions, ValidationResult}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-case class ViolationModelValidationPlugin(configName: String) extends AMFValidatePlugin {
+case class ViolationModelValidationPlugin(configName: String) extends BaseApiValidationPlugin {
```

Comment:
This change is made to that composite configs can validate aml right?

Reply:
Yep